### PR TITLE
MOB-2014 - Share addresses of MPC wallet 

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift [hunk 1]
```diff
@@ -146,6 +146,43 @@ extension WalletEntity {
...
+extension WalletEntity {
+    enum AssetsType {
+        case singleChain(BalanceTokenUIDescription)
+        case multiChain([BalanceTokenUIDescription])
+    }
+    
+    func getAssetsType() -> AssetsType {
+        if case .mpc = udWallet.type {
+            do {
+//                return .multiChain([MockEntitiesFabric.Tokens.mockEthToken(),
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift:159]: Dead? Mock shouldn't generate in prod code

File: unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -1111,6 +1111,10 @@ More tabs are coming in the next updates.";
 "IMPORT_MPC_WALLET_IN_PROGRESS_TITLE" = "Importing your\nMPC Wallet...";
 "IMPORT_MPC_WALLET_FINISHED_TITLE" = "All set, start using your MPC Wallet";
 "IMPORT_MPC_WALLET_FAILED_TITLE" = "Importing your\nMPC Wallet";
+"MPC_WALLET_SHARE_MULTI_CHAIN_DESCRIPTION" = "MPC Wallet has addresses across multiple blockchains.";
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings:1114]: All other sentences don't have `.` at the end. Is it needed here?

**Review by rommex — APPROVED** [review]
with questions

File: unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings:1114]: It is required in this sentence

**Pull request merged**